<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QMainWindow" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wireless Tools</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pbStart">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>730</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbStop">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>730</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>721</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Scanner</string>
     </attribute>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>761</width>
        <height>631</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>AP</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>STA Count</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Channel</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Encryption</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>DATAs</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>BSSID</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pbSelectDevice">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Device</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Channel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Channel : </string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_Device">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Device : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Status">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Status :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbClear">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Sniffer</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Deauth</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
